src/assets/scss/application.scss

```diff
@@ -10,149 +9,122 @@ $govuk-assets-path: "/public/";
 .govuk-heading-m,
 .hmpo-auto-submit h1,
 h2 {
-  margin-bottom: 30px;
+  @include govuk-responsive-margin(6, "bottom");
+  display: inline-block;
 }
 
 .govuk-list--bullet>li,
 .govuk-list--number>li {
-  margin-bottom: 15px;
+  @include govuk-responsive-margin(3, "bottom")
 }
 
 .govuk-inset-text {
-  margin-bottom: 30px;
+  @include govuk-responsive-margin(6, "bottom")
 }
 
 .govuk-body,
 main p,
 .govuk-body-m {
-  margin-bottom: 30px;
+  @include govuk-responsive-margin(6, "bottom")
 }
 
 .govuk-button {
-  margin-bottom: 30px;
-  margin-top: 10px;
-}
-
-.govuk-summary-list .govuk-summary-list__key {
-  color: black;
-  font-weight: bold;
-}
-
-.govuk-link:link,
-main a:link {
-  font-size: 19px;
+  @include govuk-responsive-margin(6, "bottom");
+  @include govuk-responsive-margin(2, "top");
 }
 
 .govuk-body,
 main p,
 .govuk-body-m {
-  margin-bottom: 3px;
+  @include govuk-responsive-margin(1, "bottom")
 }
 
 .hint-with-margin {
-  margin-top: 30px;
+  @include govuk-responsive-margin(6, "top")
 }
 
-
 .govuk-radios__divider {
-  margin-bottom: 15px;
+  @include govuk-responsive-margin(3, "bottom");
   margin-top: -15px;
 }
 
 .govuk-summary-list .govuk-summary-list__key {
-  color: #000;
-  font-weight: bold;
-  padding-bottom: 30px;
-  padding-top: 40px;
+  @include govuk-responsive-padding(6, "bottom");
+  @include govuk-responsive-padding(7, "top");
 }
 
 .govuk-warning-text {
-  margin-bottom: 0px;
+  @include govuk-responsive-margin(0, "bottom")
 }
 
 .govuk-error-summary {
   width: 580px;
 }
 
+#landingPageTitle {
+  @include govuk-responsive-margin(5, "bottom")
```

Comment:
**Full disclosure that I am getting up to speed with the frontend patterns in this project

What is the reason that we decided to add an ID rather than using a spacing class like `govuk-!-margin-bottom-5`? Just interested as there is only one style applied and it's an ID so I assume it's not going to be reused - do we tend to avoiding using the design systems override classes?